### DISCUSSION TITLE
How to use WindowEx.SetWindowStyle()

### DISCUSSION THREAD

**Discussion by mendedwraith** [dotMorten/WinUIEx#194] [Q&A] [answered]
Hi, I've been playing around with WindowEx theming but for the life of me I can't figure out how to use WindowEx.SetWindowStyle() with a custom style. For example, how to set Border. 

What I'm trying to achieve is a borderless window (without accent color), or perhaps a custom color, but I'm unsure if this feature is meant for that.

Edit: Well, I now have a tenuous grasp on it. Is there any way to remove the border and keep the window draggable and default title bar?

**Comment by dotMorten** [comment]
Please see the Windows documentation on window styles: https://learn.microsoft.com/en-us/windows/win32/winmsg/window-styles

**Reply by mendedwraith** [reply, marked as answer]
Thanks. I went through and played with the corresponding WindowStyle options. I haven't found a way to maintain title bar, draggability without an accent-colored border.

Are you able to confirm that you must reimplement title bar/draggability? It's not much of an issue to implement, I just try to avoid unnecessarily hacky solutions.

Edit: This is a bit of a never mind, since I was able to make the border transparent using DwmSetWindowAttribute. Though it would still be kind of nice to know the limitations of these styles.
